### ISSUE TITLE
PoE keeps crashing for me every time i start Lurker or when Lurker is running and i start PoE

### ISSUE THREAD

**Issue by HighOnBuffs** [C1rdec/Poe-Lurker#231]
Since Ritual league and 1.8.5 whenever i start Lurker my poe in windowed mode on windows 10 freezes and crashes instantly.
I reinstalled it a couple times and also restarted my PC multiple times.
The issue persists and is 100% unproduceable for me.

Its also the same outcome when i have Lurker already running and i start poe it will freeze in the first screen and crash.

Whenever i close Lurker everything works splendid again.

Please help!

**Comment by C1rdec** [comment]
Poe Lurker should not be the problem here but we can connect on discord if you need help.

**Comment by zsadan** [comment]
I have the same issue.

But only since about 10 minutes or so ago, 16:10 / 16:15 CET.
I closed PoE, started it again, the the Lurker Icon in the lower right area, PoE crashed.
Closed Lurker, started PoE, no crash. While PoE running started Lurker again, crash, same as OP.
Closed Chaos Recipe Enhancer, also accesses the PoE log, but still same issue.
If you let me know what logs you need, and where to find them / file names, I can upload them.

Thanks for your help in advance!

**Comment by C1rdec** [comment]
Can you provide the logs...

You can find the log : (C:\Users\[user]\AppData\Local\PoeLurker\app-[version]\logs)

**Comment by C1rdec** [comment]
I would like to connect with one of you guys to investigate

**Comment by zsadan** [comment]
[2021-01-22.log](https://github.com/C1rdec/Poe-Lurker/files/5857128/2021-01-22.log)

If you give me your Discord ID, we can have a chat. I will be  online for quite a few hours at least today.

**Comment by C1rdec** [comment]
Join Poe Lurker Discord : https://discord.gg/hQERv7K

**Comment by C1rdec** [comment]
For a temporary work around running POE in Admin seems to fix this issue.
I will keep you guys posted.

@macote This seems to be related to Winhook. 

Here is the error dump of Path of Exile:

```
LoadedModule[117]=C:\Users\zsada\AppData\Local\PoeLurker\app-1.8.5\winook.support\Winook.Lib.Mouse.x64.dll
State[0].Key=Transport.DoneStage1
State[0].Value=1
OsInfo[0].Key=vermaj
OsInfo[0].Value=10
OsInfo[1].Key=vermin
OsInfo[1].Value=0
OsInfo[2].Key=verbld
OsInfo[2].Value=18363
OsInfo[3].Key=ubr
OsInfo[3].Value=1316
OsInfo[4].Key=versp
OsInfo[4].Value=0
OsInfo[5].Key=arch
OsInfo[5].Value=9
OsInfo[6].Key=lcid
OsInfo[6].Value=2057
OsInfo[7].Key=geoid
OsInfo[7].Value=242
OsInfo[8].Key=sku
OsInfo[8].Value=48
OsInfo[9].Key=domain
OsInfo[9].Value=0
OsInfo[10].Key=prodsuite
OsInfo[10].Value=256
OsInfo[11].Key=ntprodtype
OsInfo[11].Value=1
OsInfo[12].Key=platid
OsInfo[12].Value=10
OsInfo[13].Key=sr
OsInfo[13].Value=0
OsInfo[14].Key=tmsi
OsInfo[14].Value=659365
OsInfo[15].Key=osinsty
OsInfo[15].Value=3
OsInfo[16].Key=iever
OsInfo[16].Value=11.1198.18362.0-11.0.220
OsInfo[17].Key=portos
OsInfo[17].Value=0
OsInfo[18].Key=ram
OsInfo[18].Value=32689
OsInfo[19].Key=svolsz
OsInfo[19].Value=930
OsInfo[20].Key=wimbt
OsInfo[20].Value=0
OsInfo[21].Key=blddt
OsInfo[21].Value=190318
OsInfo[22].Key=bldtm
OsInfo[22].Value=1202
OsInfo[23].Key=bldbrch
OsInfo[23].Value=19h1_release
OsInfo[24].Key=bldchk
OsInfo[24].Value=0
OsInfo[25].Key=wpvermaj
OsInfo[25].Value=0
OsInfo[26].Key=wpvermin
OsInfo[26].Value=0
OsInfo[27].Key=wpbuildmaj
OsInfo[27].Value=0
OsInfo[28].Key=wpbuildmin
OsInfo[28].Value=0
OsInfo[29].Key=osver
OsInfo[29].Value=10.0.18362.1316.amd64fre.19h1_release.190318-1202
OsInfo[30].Key=buildflightid
OsInfo[30].Value=C6A0BBFE-FA2B-4D86-AAB4-FB51F87B459E.1
OsInfo[31].Key=edition
OsInfo[31].Value=Professional
OsInfo[32].Key=ring
OsInfo[32].Value=Retail
OsInfo[33].Key=expid
OsInfo[34].Key=containerid
OsInfo[35].Key=containertype
OsInfo[36].Key=edu
OsInfo[36].Value=0
FriendlyEventName=Stopped working
ConsentKey=APPCRASH
```

**Comment by C1rdec** [comment]
Similar issue.

https://tweaker.userecho.com/en/communities/1/topics/388-latest-update-shows-7-taskbar-tweaker-could-not-load-library-206

**Comment by C1rdec** [comment]
Can you guys try 1.7.14.

https://github.com/C1rdec/Poe-Lurker/releases/download/v1.7.14/PoeLurker.exe

**Comment by neal5227** [comment]
I have had this issue since 1.7.15.  Can confirm that 1.7.14 does not crash PoE.

**Comment by macote** [comment]
@C1rdec , I know Winook got updated to 1.1.0 in 1.7.15. Can you create a test version with Winook 1.0.0 to see if it's related?

**Comment by C1rdec** [comment]
@macote Yessir <3

**Comment by C1rdec** [comment]
@macote It's confirmed 

@neal5227 tried 1.7.15 with a downgraded version of Winook (1.0.1) and the game does not crash any more.

**Comment by macote** [comment]
@neal5227 and others, please share information about your system in the issue I just created for this (macote/Winook#21). Windows version, language, system info, etc. Thanks!

**Comment by macote** [comment]
@C1rdec, since version 1.1.0 is a refactor of 1.0.1, I suggest that you simply replace the version used in Lurker with 1.0.1. I'm not convinced that if I can fix this in a timely manner. I might simply deprecated 1.1.0 and release a new version that reverts the code back to 1.0.1.

**Comment by C1rdec** [comment]
Fixed in 1.8.6.
Changed the Winook version to `1.0.1`.
